gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild

```diff
@@ -31,6 +31,9 @@ RDEPEND="
 	pdf? ( >=app-text/poppler-0.18 )
 	taglib? ( >=media-libs/taglib-1.4 )
 "
+
+PATCHES=( "${FILESDIR}/${P}-exiv2-0.27.1-missing-header.patch" )
```

Comment:
what difference is there to the `1.10.0` patch?

Reply:
There is actually no difference. But to be consistent, I added the same patch in a new file. Should I hard-code the file name to the original patch of version 1.10.0?

Reply:
yes please, we don't duplicate patches.

Reply:
done